src/SignalR/server/Core/src/StreamTracker.cs

```diff
@@ -16,14 +16,20 @@ namespace Microsoft.AspNetCore.SignalR
     internal class StreamTracker
     {
         private static readonly MethodInfo _buildConverterMethod = typeof(StreamTracker).GetMethods(BindingFlags.NonPublic | BindingFlags.Static).Single(m => m.Name.Equals("BuildStream"));
+        private readonly object[] _streamBufferCapacityArg;
         private ConcurrentDictionary<string, IStreamConverter> _lookup = new ConcurrentDictionary<string, IStreamConverter>();
 
+        public StreamTracker(int streamBufferCapacity = 10)
```

Comment:
remove default param, this is an internal class and we should always pass a value.